clifford/tools/g3c/__init__.py

```diff
@@ -560,16 +617,11 @@ def val_truncated_get_line_reflection_matrix(line_array, n_power):
     mat2solve = np.zeros((32, 32), dtype=np.float64)
```

Comment:
On the line above this line, `n_power should be 1 or a power of 2` is nonsense. Maybe just add a type annotations, `n_power: int)`?

Reply:
good spot